src/core/resource.h

```diff
@@ -148,6 +148,15 @@ class CResource : public CFunctionBlock, public forte::core::CFBContainer{
       return nullptr;
     }
 
+    struct InitValue {
+        CIEC_ANY* ptr;
+        std::string str;
+    };
+
+    std::vector<InitValue> mInitList;
```

Comment:
I would recommend to rename this to mInitValues, as the name "List" implies a certain implementation class/style.

Reply:
I would suggest `mInitialValues` since we also have the method `setInitialValues()`.